src/Processors/Formats/Impl/JSONEachRowRowOutputFormat.h

```diff
@@ -27,6 +27,8 @@ class JSONEachRowRowOutputFormat : public RowOutputFormatWithExceptionHandlerAda
     /// Content-Type to set when sending HTTP response.
     String getContentType() const override
     {
+        if (!settings.json.content_type.empty())
+            return settings.json.content_type;
```

Comment:
We have several others `JSON*` formats, like `JSON/JSONColumns/JSONCompactColumns/JSONCompactEachRow`. Maybe better to do it for all of them and not only for `JSONEachRow`?

Reply:
All JSON output format are supported now.